.github/workflows/check.yml

```diff
@@ -0,0 +1,21 @@
+name: "Check"
+
+on:
+  push:
+    branches:
+      - "*"
+      - "!main"
+      - "!gh-pages"
+
+jobs:
+  code:
+    name: "Check code"
+    uses: wrk-flow/reusable-workflows/.github/workflows/php-check.yml@main
+    secrets: inherit
+
+  commits:
```

Comment:
@MartinMystikJonas chceš pak nahodit auto generování changelogu? Jinak bych asi "neforcoval" conventional commits nebo i v tom vidíš přidanou hodnotu? :)

Reply:
Vycházel sem z toho vzoru, ale asi to moc nemá smysl tak to vyhodim